src/js-form-validator.ts

```diff
@@ -313,10 +313,8 @@ export class FormValidator {
       case 'phone':
         if (!allowEmpty && !value.length) return -1;
         let vals = value.split('x');
-        for (let i = 0, l = vals.length; i < l; i++)
-          vals[i] = vals[i].replace(/\D/g, '');
         value = vals.join('x').replace(/^1/, ''); // remove leading 1 if added, only for US
-        status = /^[0-9x]{7,16}$/.test(value) ? 1 : 0;
+        status = /^[0-9\-\+\s\(\)]{7,16}$/.test(value) ? 1 : 0;
```

Comment:
@Izzmo I think that isn't too difficult. For now I will add it to our form utilities as a small listener for any inputs provided and that meet the criteria discussed. Once that is in place the idea from your comment seems to be, that we could leave the current validation in place and I suppose almost disregard this pull request if we are handling it all in a different location. Does that sound right?